linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/bash/ubuntu.sh

```diff
@@ -2,4 +2,9 @@
 
 {{{ bash_instantiate_variables("var_password_pam_retry") }}}
 
-{{{ bash_ensure_pam_module_options('/etc/pam.d/common-password', 'password', 'requisite', 'pam_pwquality.so', 'retry', "$var_password_pam_retry", "$var_password_pam_retry") }}}
+{{{ bash_pam_pwquality_enable() }}}
+PAM_FILE_PATH=/usr/share/pam-configs/cac_pwquality
+if grep -qE 'pam_pwquality\.so.*retry=[^[:space:]]' "$PAM_FILE_PATH"; then
```

Comment:
We don't need to remove the retry parameter from `cac_pwquality`. The file is created in `bash_pam_pwquality_enable` macro and doesn't contain the parameter.